arangod/Aql/ExecutionPlan.h

```diff
@@ -433,6 +435,9 @@ class ExecutionPlan {
   /// prefetching on the node level should be executed.
   bool _isAsyncPrefetchEnabled{false};
 
+  // Flag there are collection nodes with forceIndexHint:true
+  unsigned _hasForcedIndexHints{false};
```

Comment:
```suggestion
bool _hasForcedIndexHints{false};
```

Reply:
Forgot to change type when moved away from counting 🤦‍♂️